bundle/manifests/operator.clusterserviceversion.yaml

```diff
@@ -34,11 +39,174 @@ spec:
     - kind: TektonConfig
       name: tektonconfigs.operator.tekton.dev
       version: v1alpha1
-  description: Install Shipwright Builds on your Kubernetes cluster
+  description: |
+    Shipwright is a framework for building container images on Kubernetes.
+
+    Read more: [https://shipwright.io](https://shipwright.io)
+
+    ## Usage
+
+    To deploy and manage [Shipwright Builds](https://github.com/shipwright-io/build) in your cluster,
+    first make sure this operator is installed and running on your cluster.
+
+    Next, create the following:
+
+    ```yaml
+    ---
+    apiVersion: operator.shipwright.io/v1alpha1
+    kind: ShipwrightBuild
+    metadata:
+      name: shipwright-operator
+    spec:
+      targetNamespace: shipwright-build
+    ```
+
+    The operator will deploy Shipwright Builds in `shipwright-build` namespace.
+    When `.spec.targetNamespace` is not set, it will deploy Shipwright Builds to the `shipwright-build` namespace.
```

Comment:
same redundancy ... though IIRC the make bundle call should fix this when you change in the other spot

Reply:
At the moment I am syncing this content manually. Syncing it with some kind of kustomize patch command would be a challenge.